src/navigation/config.js

```diff
@@ -150,6 +150,18 @@ export const settingsSheetConfig = {
   }),
 };
 
+export const pairHardwareWalletNavigatorConfig = {
+  options: ({ route: { params = {} } }) => ({
+    ...buildCoolModalConfig({
+      ...params,
+      backgroundOpacity: 1,
+      scrollEnabled: false,
```

Comment:
why?

Reply:
good question, we probs want to enable it actually.